_service/tedycab.md

```diff
@@ -0,0 +1,16 @@
+---
+name: tedyCab
+link: http://www.tedycab.com/
+description: Tous les taxis dans votre appli
+screenshot: ecran_tedycab.jpg
+featured: false
+api:
+  - le.Taxi
+---
+
+tedyCab vous permet de visualiser en temps réel les taxis disponibles autour de vous et d'en faire venir un à vous.
```

Comment:
J'ai pas vraiment d'avis tranché sur la question, fait comme tu le sens.

Le lundi 20 juin 2016, odtvince <notifications@github.com> a écrit :

> In _service/tedycab.md
> <https://github.com/sgmap/api.gouv.fr/pull/112#discussion_r67719859>:
>
> > @@ -0,0 +1,16 @@
> > +---
> > +name: tedyCab
> > +link: http://www.tedycab.com/
> > +description: Tous les taxis dans votre appli
> > +screenshot: ecran_tedycab.jpg
> > +featured: false
> > +api:
> > +  - le.Taxi
> > +---
> > +
> > +tedyCab vous permet de visualiser en temps réel les taxis disponibles autour de vous et d'en faire venir un à vous.
>
> C'est leur graphie officielle : http://www.tedycab.com/
> J'ai choisi de la respecter, mais ça se discute effectivement.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/sgmap/api.gouv.fr/pull/112/files/7c6f8861e8b9ec8f6ad66d4a101001a4bf3b25e4#r67719859>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFlwwe0jt4kJe9UU3GPunehdd5BwfmU5ks5qNr5pgaJpZM4I5jcv>
> .
>


-- 
Thibaut Géry